pkg/cmd/operator2/cmd.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 const (
-	componentName      = "cluster-osin-operator2"
-	componentNamespace = "openshift-core-operators"
+	componentName      = "cluster-authentication-operator2"
+	componentNamespace = "openshift-authentication-operator"
```

Comment:
cite requirements

Reply:
I've placed naming requirements from email thread in commit msg for reference.